package.json

```diff
@@ -43,6 +43,7 @@
         "less-middleware": "^2.0.1",
         "moment": "^2.10.3",
         "morgan": "^1.5.1",
+        "mkdirp": "^0.5.1",
```

Comment:
No need for this extra dependency. We already have this functionality in one of our other libraries, fs-extra.